docs/src/components/MainSectionCard.js

```diff
@@ -76,8 +76,33 @@ const MainSectionCard = ({
     );
   };
 
+  const TitleAndDescription = (
+    <Box
+      marginTop={borderStyle ? 4 : 3}
+      marginBottom={cardSize === 'lg' ? 4 : 0}
+      dangerouslySetInlineStyle={{
+        __style: { borderTop: borderStyle },
+      }}
+    >
+      {(title || type !== 'info') && (
+        <Box paddingY={1}>
+          <Text weight="bold" color={TYPE_TO_COLOR[type]}>
+            {cardTitle || type.charAt(0).toUpperCase() + type.slice(1)}
```

Comment:
Don't we have a util for capitalization now?

Reply:
We didn't, but I made a new file for us to use w/i the Docs components